llvm/test/Feature/Repo/repo_dependencies_noInline.ll

```diff
@@ -0,0 +1,22 @@
+; Check that the GOs' dependencies and contributions list.
```

Comment:
This seems like a half-written thought! What is this test checking for?

Reply:
It checks that the `test`'s dependencies which only include the `Z` not include `setto` since `setto` is a no-inline function.
Added the comment to the test.